Sources_v3/StreamChatUI/Common Views/CreateNewChannelButton.swift

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class CreateNewChannelButton: UIButton, AppearanceSetting {
+    // MARK: - Overrides
+    
+    override open var intrinsicContentSize: CGSize {
+        .init(width: 44, height: 44)
+    }
+    
+    // MARK: - Init
+    
+    override public init(frame: CGRect) {
+        super.init(frame: frame)
+        applyDefaultAppearance()
+    }
+    
+    public required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        applyDefaultAppearance()
+    }
+    
+    // MARK: - AppearanceSetting
+
+    public static func initialAppearanceSetup(_ button: CreateNewChannelButton) {
+        button.setImage(UIImage(named: "icn_new_chat", in: Bundle(for: Self.self), compatibleWith: nil), for: .normal)
```

Comment:
Two things here:
- ideally, we'd have these images defined as constants like `UIImage.stream.iconNewChat`
- are we sure loading the images like this is safe when the SDK is imported via SPM?

Reply:
The way resources are exposed from `StreamChatUI` feels like big separate issue and requires some time. Using some tool that generates the way images will be accessed (`UIImage.stream.iconNewChat`) also feels more reasonable than handling it manually.

Couple of opened PRs require `.xcassets` in `StreamChatUI` so this can also be part of the `resource backbone` PR.

Reply:
Agreed with @evsaev , we need a resources backbone PR. Using resource bundles (not asset catalogs) is supported by all dependency managers

Reply:
@VojtaStavik @b-onc added as a subtask to [Backbone for UI SDK](https://stream-io.atlassian.net/secure/RapidBoard.jspa?rapidView=61&projectKey=CIS&modal=detail&selectedIssue=CIS-409)